### PR TITLE
RavenDB-18360 When setup fails due to DNS propagation link to the dom…

### PR DESCRIPTION
**Pull request by Danielle9897**
…ain in question directly

### Issue link
https://issues.hibernatingrhinos.com/issue/RavenDB-18360

### Additional description
Specify the exact google API that was used to resolve the hostname

### Type of change
- Usability

### How risky is the change?
- Low 

### Backward compatibility
- Non breaking change
### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing 
- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

**Comment by Danielle9897** [comment]
@gregolsky 
I have used `https://dns.google/resolve?name=` since I saw that was the exact API used
Please confirm whether to keep this - or use the `/query` instead

**Comment by gregolsky** [comment]
`/query` shows a nice user friendly result and present the same data. Can we use `/query`?

**Comment by Danielle9897** [comment]
done

**Review by ml054 — APPROVED** [review]

File: src/Raven.Server/Commercial/SetupManager.cs [hunk 1]
```diff
@@ -2030,30 +2029,33 @@ private static async Task AssertDnsUpdatedSuccessfully(string serverUrl, IPEndPo
                 var expectedIps = expectedAddresses.Select(address => address.Address.ToString()).ToHashSet();
 
                 var hostname = new Uri(serverUrl).Host;
+                const string googleDnsApi = "https://dns.google.com";
+                const string googleDnsQueyrApi = "https://dns.google/query?name=";
```
Comment (gregolsky) [inline src/Raven.Server/Commercial/SetupManager.cs]: Queyr -> Query

**Review by gregolsky — COMMENTED** [review]
just that typo I guess

File: src/Raven.Server/Commercial/SetupManager.cs (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Server/Commercial/SetupManager.cs]: right... forced pushed the fix

**Review by gregolsky — APPROVED** [review]

**Pull request merged**